svo (setup, misc, empty, funnies, dor).xml

```diff
@@ -2840,9 +2840,7 @@ end
 
 empty.eat_bloodroot = function()
   svo.lostbal_herb()
-  svo.rmaff('paralysis')
-
-  if not affs.stain then svo.rmaff('slickness') end
+  svo.rmaff('paralysis', 'slickness')
```

Comment:
rmaff function only accepts one arg
just call it twice once for para and once for slick

Reply:
oh, rip, i thought i copied this from somewhere but i guess it was a different function 🤦‍♂️

Reply:
I updated this to an array of the affs, which I think is right based on existing (which is what I was trying to copy but missed that it should be an array):

https://github.com/MediaResAchaea/svof/blob/ae640b2d4ea8214b84dc26d44558c2023486fe9d/svo%20(setup%2C%20misc%2C%20empty%2C%20funnies%2C%20dor).xml#L2921